internal/torcx.go

```diff
@@ -206,13 +180,20 @@ func (a *App) UseAddon(name string, reference string) error {
 }
 
 // TorcxGC removes versioned stores that we know we won't need.
+// It takes a list of OS versions to keep
```

Comment:
Can we only pass the minimum of the versions, and the remove user stores which have valid semver-names and sort less than the minimum? This is to make sure we don't nuke something we didn't create, by mistake.